code/modules/clothing/suits/jobs.dm

```diff
@@ -151,6 +156,7 @@
 	desc = "A sterile blue surgical apron."
 	icon_state = "surgical"
 	allowed = list(/obj/item/scalpel, /obj/item/surgical_drapes, /obj/item/cautery, /obj/item/hemostat, /obj/item/retractor)
+	species_exception = list(/datum/species/golem)
```

Comment:
this is a subtpe of normal aprons, so I don't think that you need to set the value of species_exception here again

Reply:
When I realized that I swear I fixed up all the aprons. I'll go clean this up.